lib/rubygems/commands/query_command.rb

```diff
@@ -129,55 +137,55 @@ def display_header(type)
 
   #Guts of original execute
   def show_gems(name, prerelease)
```

Comment:
You have extracted `prerelease` to a method, so this method argument is now unused.